ansible/ncn_nodes.yml

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ansible-playbook
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# This is a top level play for all NCN nodes that encorporates all 3 kinds of
+# NCN node (management, storage, worker) as well as their Image counterparts.
+# The three sepearate top level plays are marked for deprecation via a debug
+# notice.
+- hosts:
+   - Management_Master
```

Comment:
Is the idea for all of these tasks that if they are not specifically guarded by either a '&cfs_image' or '!cfs_image', then they are meant to run in both contexts? This is a generic comment meant for the entire PR, not just this section.

Reply:
as mentioned in one of the tickets where you asked the same thing, yes.

Flagging a playbook as something that needs to be done in image only is not a valid use case. We do not want to require people reboot their nodes to get updated content deployed to nodes. Therefor, almost all tasks must run during node personalization, but _some_ tasks can run ahead of time.